src/Features/Core/Portable/Completion/Providers/AbstractMemberInsertingCompletionProvider.cs

```diff
@@ -80,9 +81,11 @@ private async Task<Document> DetermineNewDocumentAsync(
         var tree = await document.GetRequiredSyntaxTreeAsync(cancellationToken).ConfigureAwait(false);
         var token = GetToken(completionItem, tree, cancellationToken);
         var annotatedRoot = tree.GetRoot(cancellationToken).ReplaceToken(token, token.WithAdditionalAnnotations(_otherAnnotation));
-        // Make sure the new document is frozen before we try to get the semantic model. This is to 
-        // avoid trigger source generator, which is expensive and not needed for calculating the change.
-        document = document.WithSyntaxRoot(annotatedRoot).WithFrozenPartialSemantics(cancellationToken);
+
+        // Make sure the new document is frozen before we try to get the semantic model. This is to avoid trigger source
+        // generator, which is expensive and not needed for calculating the change.  Pass in 'forceFreeze: true' to
+        // ensure all further transformations we make do not run generators either.
+        document = document.WithSyntaxRoot(annotatedRoot).WithFrozenPartialSemantics(forceFreeze: true, cancellationToken);
```

Comment:
here's an example of where we want 'forceFreeze: true'.  We are making several transformation passes, and we don't want generators running between each phase.